src/Microsoft.SqlTools.Dmp.Contracts/Hosting/HostingErrorNotification.cs

```diff
@@ -0,0 +1,28 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Dmp.Contracts;
+
+namespace Microsoft.SqlTools.Dmp.Contracts.Hosting
+{
+    /// <summary>
+    /// Parameters to be used for reporting hosting-level errors, such as protocol violations
+    /// </summary>
+    public class HostingErrorParams
+    {
+        /// <summary>
+        /// The message of the error
+        /// </summary>
+        public string Message { get; set; }
+    }
+
+    public class HostingErrorNotification
+    {
+        public static readonly 
+            EventType<HostingErrorParams> Type =
+            EventType<HostingErrorParams>.Create("hosting/error");
```

Comment:
What will this be replacing? Will the DMP need updating to support this?

Reply:
Nope, this was already in the hosting project, just with a slightly different class name. Check \src\Microsoft.SqlTools.Hosting\Hosting\Protocol\MessageDispatcher.cs:221 for one of the old usages of it.